botx/models/system_events/cts_logout.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from typing import Any, Dict
+from uuid import UUID
+
+from pydantic import Field
+
+from botx.models.api_base import VerifiedPayloadBaseModel
+from botx.models.base_command import (
+    BaseBotAPIContext,
+    BotAPIBaseCommand,
+    BotCommandBase,
+)
+from botx.models.enums import BotAPICommandTypes
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+@dataclass
+class CTSLogoutEvent(BotCommandBase):
+    """Event `system:cts_logout`.
+
+    Attributes:
+        user_huid: user ID.
+        cts_id: ID of the server the user is logged out from.
+    """
+
+    user_huid: UUID
+    cts_id: UUID
+
+
+class BotAPICTSLogoutData(VerifiedPayloadBaseModel):
+    user_huid: UUID
+    cts_id: UUID
+
+
+class BotAPICTSLogoutPayload(VerifiedPayloadBaseModel):
+    body: Literal["system:cts_logout"] = "system:cts_logout"
+    command_type: Literal[BotAPICommandTypes.SYSTEM]
+    data: BotAPICTSLogoutData
+    metadata: Dict[str, Any]
```

Comment:
Drop `metadata` from check, you don't use it.

Reply:
done